src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const JobsTrainingButtons = ({ setCardsArr, setSearchCards }) => {
+  const { t } = useTranslation();
+  const [isDisabled, setIsDisabled] = useState(true);
+  const buttonText = ["Job Opportunities", "Trainings / Workshops"];
+
+  const selectOpportunityButton = () => {
+    if (isDisabled === true) {
+      setSearchCards([...t(`home.trainings`, { returnObjects: true })]);
+      setCardsArr([...t(`home.trainings`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    } else {
+      setSearchCards([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setCardsArr([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    }
+  };
+
+  const opportunityListBtn = buttonText.map((text, index) => {
+    return (
+      <button
+        key={text}
+        type="button"
+        className="opportunityListBtn"
+        onClick={() => {
+          selectOpportunityButton();
+        }}
+        disabled={index === 0 ? isDisabled : !isDisabled}
```

Comment:
what if we add another button to the list, what would happen? I do not think that would work

Reply:
There will be no other button actually it's just jobs and trainings. 

This only disables the already selected button to indicate on the screen that visitors that right now you are seeing the jobs cards. If I don't add it both button look the same and it doesn't show where the cards belong to. 

 We can discuss in after the sprint furthermore.